rules/CodingStyle/ClassNameImport/UseImportsTraverser.php

```diff
@@ -25,67 +25,39 @@ public function __construct(
 
     /**
      * @param Stmt[] $stmts
-     * @param callable(UseUse $useUse, string $name): void $callable
+     * @param callable(int $useType, UseUse $useUse, string $name): void $callable
```

Comment:
Here also enum could be I think

Reply:
The constants used are from PHPParser.

Reply:
By "enum" I mean the PHPStan docblock style :)
https://github.com/rectorphp/rector-src/pull/4998/files#r1322839985

Reply:
thats only possible after https://github.com/nikic/PHP-Parser/pull/945 is merged.

typing this before the PR is merged would lead to

```
 ------ ---------------------------------------------------------------------------------------------------------------------------
  Line   rules/CodingStyle/ClassNameImport/UseImportsTraverser.php
 ------ ---------------------------------------------------------------------------------------------------------------------------
  47     Parameter #1 $useType of callable callable(0|1|2|3, PhpParser\Node\Stmt\UseUse, string): void expects 0|1|2|3, int given.
  70     Parameter #1 $useType of callable callable(0|1|2|3, PhpParser\Node\Stmt\UseUse, string): void expects 0|1|2|3, int given.
 ------ ---------------------------------------------------------------------------------------------------------------------------
 ```

Reply:
Feel free to add it here and ignore these errors, as it improves reading of our code.

Reply:
here you are

Reply:
Thanks :+1: